src/routes.jsx

```diff
@@ -1,35 +1,39 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
 import PropTypes from 'prop-types';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
 import Home from './Home';
-import AndroidPage from './android';
-import NimbledroidGraphPage from './nimbledroid/NimbledroidGraphPage';
-import Quantum from './quantum/Quantum';
-import TP6 from './quantum/TP6';
-import TP6M from './android/TP6m';
 import Power from './power';
+import NoMatch from './NotFound';
 import Playback from './playback/playback';
 import PlaybackDetails from './playback/details';
-import Subtests from './quantum/subtests';
 
-const NoMatch = () => <div>404</div>;
+const AndroidPage = lazy(() => import(/* webpackChunkName: "andriod" */ './android'));
+const NimbledroidGraphPage = lazy(() => import(/* webpackChunkName: "nimbledroid" */ './nimbledroid/NimbledroidGraphPage'));
+const Quantum = lazy(() => import(/* webpackChunkName: "quantum" */ './quantum/Quantum'));
+const TP6 = lazy(() => import(/* webpackChunkName: "quantum-tp6" */ './quantum/TP6'));
+const TP6M = lazy(() => import(/* webpackChunkName: "andriod-tp6m" */ './android/TP6m'));
+const Subtests = lazy(() => import(/* webpackChunkName: "quantum-subtests" */ './quantum/subtests'));
```

Comment:
I still do not think we need a split by page.  We only need to split out large logic chunks, like the charting libraries.

Reply:
...but this is not a blocker